kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -70,16 +67,47 @@ object JazzerLauncher {
         }
 
         libFuzzerArgs += currentCorpus.toString()
-        libFuzzerArgs += "-max_total_time=${config.maxSingleTargetFuzzTime.inWholeSeconds}"
         libFuzzerArgs += "-rss_limit_mb=${jazzerConfig.libFuzzerRssLimit}"
+        libFuzzerArgs += "-artifact_prefix=${reproducerPath.toAbsolutePath()}/"
+
+        var keepGoing =
+            if (config.runModes.contains(RunMode.REGRESSION)) reproducerPath.listDirectoryEntries("crash-*").size.toLong() else 0
```

Comment:
Very long and unreadable line.
```kotlin
var keepGoing = when (RunMode.REGRESSION) {
        in config.runModes -> reproducerPath.listDirectoryEntries("crash-*").size.toLong()
        else -> 
}
```
Is much better

Reply:
Also, it makes sence to extract `listDirectoryEntries("crash-*")` into a method like `Path.getCrashes`. You use it multiple times. Plus, "*crash-*" are not the only crashes, there are also "timeout-*" and "slow-input-*", which we may considere in future

Reply:
Fixed